db/schema.rb

```diff
@@ -177,10 +177,10 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "job_title"
-    t.string "has_no_deductions"
     t.bigint "amount"
     t.string "frequency"
     t.jsonb "metadata", default: {}, null: false
+    t.string "has_no_deductions"
```

Comment:
Does this need to be deleted?

Reply:
Its just in a different order in my schema.